showcase/app/templates/index.hbs

```diff
@@ -314,5 +308,19 @@
         </LinkTo>
       </li>
     </ol>
+
+    <Shw::Text::H2>Deprioritized</Shw::Text::H2>
```

Comment:
@cristiano recommended creating a new section like this. I can change it back and add a notice.

Reply:
I saw the suggestion in the memo afterwards. I wouldn't want to add more to your work, since it's done already let's leave it as is, but maybe we can use 'unpublished' rather than 'deprioritized'

Reply:
I changed it back because I think that will make it easier if we publish the components in the future.